packages/env/test/config.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
```

Comment:
I think there's technically some value in this line being here? It's otherwise assumed by default in [ES modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Strict_mode_for_modules) (and [emulated by Babel](https://github.com/babel/babel/issues/1844#issuecomment-115574757)), but neither of those apply in this non-transpiled CommonJS code.

Reply:
We don't use `use strict` in other unit tests, and all those files are transpiled so I guess we could use import statements to make it more obvious.

Reply:
I am anxiously awaiting [October](https://nodejs.org/en/about/releases/) when we can finally start using ["type": "module"](https://nodejs.org/api/esm.html#esm_package_json_type_field) in these utilities 😄